bazel/repository_locations.bzl

```diff
@@ -120,13 +120,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         # 2. Open https://chromium.googlesource.com/chromium/src/+/refs/tags/<current_version>/DEPS and note <boringssl_revision>.
         # 3. Find a commit in BoringSSL's "master-with-bazel" branch that merges <boringssl_revision>.
         #
-        # chromium-112.0.5615.39 (linux/beta)
-        version = "88d7a40bd06a34da6ee0d985545755199d047258",
-        sha256 = "1e759891e168c5957f2f4d519929e2b4cef9303b7cf2049601081f4fca95bf21",
+        # chromium-118.0.5993.54 (linux/beta)
+        version = "45cf810dbdbd767f09f8cb0b0fcccd342c39041f",
+        sha256 = "f1f421738e9ba39dd88daf8cf3096ddba9c53e2b6b41b32fff5a3ff82f4cd162",
         strip_prefix = "boringssl-{version}",
         urls = ["https://github.com/google/boringssl/archive/{version}.tar.gz"],
         use_category = ["controlplane", "dataplane_core"],
-        release_date = "2023-02-14",
+        release_date = "2023-10-04",
```

Comment:
The date of this commit, according to https://boringssl.googlesource.com/boringssl/+/45cf810dbdbd767f09f8cb0b0fcccd342c39041f, is `Aug 28 21:44:34 2023`. Please put that date here.

Reply:
Done, thanks.